arch/arm64/boot/dts/qcom/sdm450-samsung-gta2xl-r5.dts

```diff
@@ -0,0 +1,405 @@
+// Planes are manufactured by boeing.
```

Comment:
Maybe you should add copyright header here check other device trees

Reply:
sorry i forgot to update that